src/main/java/com/school/project/controller/UserController.java

```diff
@@ -21,9 +23,20 @@
     @Autowired
     UserService userService;
 
+    @Autowired
+    UserAccountService userAccountService;
+
     @PostMapping("/users")
-    public void createUser(@RequestBody UserDto userDto) {
-        userService.create(convertUserDtoToUser(userDto));
+    public ResponseEntity<UserDto> createUser(@RequestBody UserDto userDto) {
+        return ResponseEntity.ok()
+                .body(convertUserToUserDto(userService
+                        .create(convertUserDtoToUser(userDto))));
+    }
+
+    @PutMapping("/users/userAccount/{id}")
+    public ResponseEntity updateUserAccountType(@PathVariable long id, @RequestBody UserAccountType userAccountType){
+        userAccountService.getUserAccountById(id).setAccountRole(userAccountType);
```

Comment:
are you sure you will get userAccount id here?

Reply:
For your interest, we won't have UserAccountId on FE ever, no reason for this. We even don't have DTO for user account. All what you got is UserId